functions.php

```diff
@@ -1,6 +1,8 @@
-<?php
+<?php	
+
  
 register_sidebar(array(
```

Comment:
Any calls to `register_sidebar()` here should be wrapped inside a function that is attached to the `widgets_init` action. See https://github.com/washingtonstateuniversity/WSUWP-spine-parent-theme/blob/53b99e76c2f1049245dc4c5c198b48b2ffe1b93c/functions.php#L428-L442 as an example.

If these are not attached to an action, then the functions fire on every single page load, whether that page load supports side bars or not. Set `WP_DEBUG` to `true` in your local `wp-config.php` to see the errors this generates.